### ISSUE TITLE
Some questions about experiments

### ISSUE THREAD

**Issue by Pei233** [XiangZ-0/EVDI#1]
Hi, authors.

Thank you for the excellent work about event-based deblurring. There are some questions about the comparison experiments.

1. I find some differences in the quantitative results among existing methods. How do you obtain these results, such as LEDVDI, eSL-Net, and RED? Did you retrain or implement the compared methods on your datasets? 
2. I saw _Sequence prediction comparison_  of Table 1 in 'Motion Deblurring with Real Events.' But I am not sure it has the same meaning as Table 2 in your paper?

**Comment by XiangZ-0** [comment]
Hello Pei233,

Thanks for your interest in our work! 
1. For the learning-based methods, we use the official codes and models provided by the authors for comparison. The differences may be caused by some details in blur and event synthesis such as blur degree, image resolution, and frame rate conversion. In our experiments, all comparisons are conducted under the same dataset and configuration.
2. They don't mean the same. Actually, the _Sequence prediction comparison_ in 'Motion Deblurring with Real Events.'  is similar to the setting of our deblurring experiment shown in Table 1. Table 2 in our paper shows the quantitative results of the frame interpolation task, with the experimental details presented in Section 5.3.
Hope it helps with your research :-)

Best,
Xiang

**Comment by Pei233** [comment]
Thank you for the reply.

For Q1, thus, you evaluate the existing pre-trained models on your datasets. 
For Q2, another question is, what's the difference between the _Single frame prediction comparison_ and _Sequence prediction comparison_? And Table 1 in your paper belongs to the single or sequence prediction? According to the details in Section 5.2., it seems to belong to the sequence prediction?

BTW, are you convenient for sharing the codes of experimental data generation? For example, the generation of the .npz file from the HQF dataset (.bag format).

Thanks.

**Comment by XiangZ-0** [comment]
Yes, the results in Table 1 of our paper are obtained under _Sequence prediction_. In my understanding, _Single frame prediction_ means that only one latent frame per blurry input (e.g., the latent frame at the middle of the exposure time of the corresponding blurry frame) is used to evaluate the performance, while _Sequence prediction_ means that a sequence of restored latent frames are used for evaluation, which is more challenging since it requires the deblurring method to handle temporal ambiguity. 
For the data generation, we may release the related code in the future. You can also follow the instructions in our paper (Section 5) to construct the dataset.
